autoconfigure/src/main/java/zipkin2/autoconfigure/storage/kafka/ZipkinKafkaStorageProperties.java

```diff
@@ -28,7 +28,7 @@
 
   private String bootstrapServers;
 
-  private Long traceTtlCheckInterval;
+  private Long traceFlushInterval;
```

Comment:
did you rename the right property?

Reply:
pushed to early, check latest changes :)